Sources/ParseSwift/Types/ParseError.swift

```diff
@@ -406,3 +406,90 @@ extension ParseError: CustomStringConvertible {
         debugDescription
     }
 }
+
+// MARK: Compare Errors
+extension Error {
+    /**
+     Validates if the given ParseError Code is equal to the error.
+     
+    **Example use case:**
+    ~~~
+    if error.containedInParseError(.objectNotFound)  {
+        //Do stuff
+    }
+    ~~~
+     - parameter errorCode: The ParseError code you want to compare to
+     */
+    public func equalsToParseError(_ errorCode: ParseError.Code) -> Bool {
+        guard let parseError = self as? ParseError else {
+            return false
+        }
+        return parseError.code == errorCode
+    }
+
+    /**
+     Returns an optional ParseError if the given ParseError Code is equal to the error.
+     
+    **Example use case:**
+    ~~~
+    if let parseError = error.equalsToParseError(.objectNotFound)  {
+        print(parseError.description)
+    }
+    ~~~
+     - parameter errorCode: The ParseError code you want to compare to
+     
+     - returns: An optional ParseError
+     */
+    public func equalsToParseError(_ errorCode: ParseError.Code) -> ParseError? {
```

Comment:
See my other comment about ending method names. You will want to make all changes to comments and playgrounds.